modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/info.scala

```diff
@@ -0,0 +1,15 @@
+package org.tessellation.sdk.infrastructure.consensus
+
+import org.tessellation.schema.peer.PeerInfo
+
+import derevo.circe.magnolia._
+import derevo.derive
+
+object info {
+
+  @derive(encoder)
+  case class ConsensusInfo[Key](
+    key: Key,
+    peersInfo: Set[PeerInfo]
```

Comment:
Why not "peers"? IMO it's more intuitive as it's just a DTO for http api. You query for /peers. PeerInfo is just our internal name, to distinguish it from Peer class.